### ISSUE TITLE
In address form, Country should be placed on top, so customer can fill state/region/zip code easily

### ISSUE THREAD

**Issue by HartLarsson** [PrestaShop/PrestaShop#21451]
If you want to facilitate the state/region/zip fill by the customer  during checkout, the Country field must be placed in top position in adress form like:

![image](https://user-images.githubusercontent.com/16102074/96006808-26675700-0e3e-11eb-99d8-f1750c91dd6e.png)

In this way the customer choose its country and automatically all the ZIP, State/Region are populated correctly.

The main problem is to change position to Country field you can operate in the Adress layout that will change also how the customer adress is printed in backoffice and in all documents and email.

Should be better that the form layout must be formatted separately from the adress format.

**Comment by florine2623** [comment]
Hi @HartLarsson ,
I will ping our product team, please stay tuned !
Thanks!

**Comment by florine2623** [comment]
Hi @PrestaShop/product-team ,
Do you think @HartLarsson 's idea is relevant for the address format ?
Thanks!

**Comment by MatShir** [comment]
It is clearly relevant. It will avoid the merchant to change their information if the selected country changes the form. 
But I don't understand why it should change the order in the displayed addresses. We only have to change the order in the Form not how the complete address is displayed afterward, no?

**Comment by HartLarsson** [comment]
> It is clearly relevant. It will avoid the merchant to change their information if the selected country changes the form.
> But I don't understand why it should change the order in the displayed addresses. We only have to change the order in the Form not how the complete address is displayed afterward, no?

I totally agree with you, but actually the only way to change form field order is to change adress fields order that is not good at all.

**Comment by MatShir** [comment]
I see, thanks for your clarification

**Comment by HartLarsson** [comment]
any solution? thanks
